From 47141bbc2fa1003499ea5151bc78a6fe6305ed89 Mon Sep 17 00:00:00 2001
From: Li Lingfeng <lilingfeng3@huawei.com>
Date: Fri, 6 Jan 2023 12:16:48 +0800
Subject: [PATCH 005/905] [Huawei] fix kabi broken due to may_pollfree

Offering: HULK
hulk inclusion
category: bugfix
bugzilla: 188217

--------------------------------

Commit 0845c5803f3f("[Backport] io_uring: disable polling pollfree
files") adds a new member in file_operations, so we need to fix
kabi broken problem.

Signed-off-by: Li Lingfeng <lilingfeng3@huawei.com>
---
 include/linux/fs.h | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/include/linux/fs.h b/include/linux/fs.h
index 991f044229e5..84878a9c5830 100644
--- a/include/linux/fs.h
+++ b/include/linux/fs.h
@@ -1885,9 +1885,8 @@ struct file_operations {
 				   struct file *file_out, loff_t pos_out,
 				   loff_t len, unsigned int remap_flags);
 	int (*fadvise)(struct file *, loff_t, loff_t, int);
-	bool may_pollfree;
 
-	KABI_RESERVE(1)
+	KABI_USE(1, bool may_pollfree)
 	KABI_RESERVE(2)
 	KABI_RESERVE(3)
 	KABI_RESERVE(4)
-- 
2.17.1

